src/operations/expansions/inside.js

```diff
@@ -0,0 +1,49 @@
+/* Determine if the given points are inside the polygon.
+ *
+ * @param {Array} points - a list of points, where each point is an array with X and Y values
+ * @param {Array} polygon - a list of points, where each point is an array with X and Y values
+ * @return {Integer} 1 if all points are inside, 0 if some or none are inside
+ */
+const arePointsInside = (points, polygon) => {
+  let sum = points.reduce((acc, point) => {
+    return acc + isPointInside(point, polygon)
+  }, 0)
+  return sum === polygon.length ? 1 : 0
+}
+
+/* Determine if the given point is inside the polygon.
+ *
+ * @see http://paulbourke.net/geometry/polygonmesh/
+ * @param {Array} point - an array with X and Y values
+ * @param {Array} polygon - a list of points, where each point is an array with X and Y values
+ * @return {Integer} 1 if the point is inside, 0 if outside
+ */
+const isPointInside = (point, polygon) => {
```

Comment:
these two are great, but I think they belong in a more generic place, for reuse (over the years I had a need for these a lot in designs

Reply:
Sure. Do you have some suggestions? I’m thinking of two places...
- geometry/poly3
- utils

This is pretty generic. This polygon is defined as an ordered list of points.

Reply:
hmm I agree about generic, but 'utils' seems too generic: this will only work for 2D points, so I would
- put it under geom2 since we do not have 2d polygons as data structures? 
- rename the file to a more clear arePointsInsidePoly2.js

Reply:
@kaosat-dev how about creating a poly2 data structure? The only uses now are:
- measureArea (now called area() is math utils)
- arePointsInside

It seems kind of a waste but really these functions would better align as part of this data structure.

Reply:
ok, agreed, not perfect, but coherent, and we can always change later !

Reply:
Done. Create poly2 geometry with minimal functionality. Updated all expansion functions to use poly2 for mearsureArea and arePointsInside.